resources/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -0,0 +1,53 @@
+deployment:
+  replicaCount: 1
+  image:
+    pullPolicy: IfNotPresent
+  securityContext:
+    runAsUser: 2000
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+broker:
+  port: "8080"
+  username: "broker"
+  # password can be set manually by user. If empty then it will be randomized.
+  password: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+provisioner:
+  URL: "http://compass-provisioner.compass-system.svc.cluster.local:3000/graphql"
+  gardener:
+    # name of the secret with kubeconfig to the gardener cluster
+    secretName: "gardener"
+    # name of the gardener project
+    projectName: "gopher"
```

Comment:
`gopher` project name seems to be not the production configuration

Reply:
We plan to make this field configurable in the future when the whole scenario with our broker will work.